signer/keydbstore/rethink_keydbstore.go

```diff
@@ -77,7 +79,7 @@ func rdbPrivateKeyFromJSON(data []byte) (interface{}, error) {
 // PrivateKeysRethinkTable is the table definition for notary signer's key information
 var PrivateKeysRethinkTable = rethinkdb.Table{
 	Name:             RDBPrivateKey{}.TableName(),
-	PrimaryKey:       RDBPrivateKey{}.KeyID,
+	PrimaryKey:       "key_id",
```

Comment:
`RDBPrivateKey{}`.KeyID was just the empty string, because it initialized the empty struct and so that value is just the zero value.  If we wanted to declare a constant and use it to set the field value, and hence use the constant here, or if we wanted to pull out the field value, I think we're going to need some reflection to do so, or better yet:  https://github.com/oleiade/reflections.

If we wanted to add that though, maybe it can be a different PR?